docs/docusaurus.config.js

```diff
@@ -0,0 +1,112 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: "Open Assistant",
+  tagline: "Build the assistant of the future!",
+  url: "https://andrewm4894.github.io",
```

Comment:
this probably needs to become the main url :)

Reply:
Yep, I can change all that in one last commit now before we merge this. Was just leaving like that so my site was working. 

Will make follow up commit to this branch to replace all the andrewm4894 with LAION-AI